src/js/AccessibilityUtils.js

```diff
@@ -549,10 +549,12 @@ axs.utils.isNativelyDisableable = function(element) {
  * This element may or may not be effectively disabled in practice as this is dependent on implementation.
  *
  * @param {Element} element An element to check.
+ * @param {boolean} self If true only consider the element itself.
```

Comment:
This param name is a little confusing, given `self` is often used to alias `this`. Perhaps something like `checkForInherited` (sense inverted)?

Also, this only limits the scope for the `aria-disabled` check, and doesn't prevent walking up the parent chain for native `disabled`. Is that deliberate? Perhaps early-out the for loop if this is true (or false if the sense is inverted, obviously)?